assets/src/edit-story/components/panels/shared/flipControls.js

```diff
@@ -31,30 +32,61 @@ import { ReactComponent as FlipHorizontal } from '../../../icons/flip_horizontal
 import { ReactComponent as FlipVertical } from '../../../icons/flip_vertical.svg';
 import Toggle from '../../form/toggle';
 
-function FlipControls({ value, onChange }) {
+const styledToggle = css`
+  width: 36px;
+  height: 36px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  float: left;
```

Comment:
Let's try to avoid floats.

IMO we should be able to set 

```css
display: flex;
justify-content: left;
align-items: flex-start;
```

on `ControlsContainer`

Then, there's no real need for a separate `stylesToggle` variable. It can be inlined in `ToggleContainer`:

```
const ToggleContainer = styled.div`
  width: 36px;
  height: 36px;
  display: flex;
  justify-content: center;
  align-items: center;
`
```

Reply:
Done.